Gemfile

```diff
@@ -52,6 +52,8 @@ gem 'selenium-webdriver', '~> 3.142.3'
 
 gem 'rubyzip', '~> 1.3.0'
 
+gem 'whenever'
```

Comment:
Let's actually move the use of this gem. In production (on Heroku), we have a task scheduler.